src/app/index.ts

```diff
@@ -254,15 +255,16 @@ module.exports = yo.extend({
         let language = this.project.scriptType === typescript && !this.project.isExcelFunctionsProject  ? 'ts' : 'js';
         const starterCode = generateStarterCode(this.project.host);
         const templateFills = Object.assign({}, this.project, starterCode);
+        let jsonData = new projectsJsonData(this.templatePath()); 
 
         this._projectCreationMessage();
         
         if (this.project.isExcelFunctionsProject)

```

Comment:
While this is OK for now, you could also structure the code to see if there is a repository for the project type. If not (because null is returned), then it could fall back to the existing template logic in the else clause. That would make it for the next project, we just need to update the projects json with a repository path to use it and not rely on a special case in code for custom functions project.

Reply:
I will add logic to fall back to existing template if getProjectTemplateRepository returns undefined